elastic/tests/test_elastic.py

```diff
@@ -202,6 +203,21 @@ def test_index_metrics(dd_environment, aggregator, instance, cluster_tags):
         aggregator.assert_metric(m_name, tags=cluster_tags + ['index_name:testindex'])
 
 
+@pytest.mark.integration
+def test_cat_allocation_metrics(dd_environment, aggregator, instance, cluster_tags):
+    instance['cat_allocation_stats'] = True
+    elastic_check = ESCheck('elastic', {}, instances=[instance])
+    cat_allocation_metrics = {}
+    cat_allocation_metrics.update(CAT_ALLOCATION_METRICS)
+    es_version = elastic_check._get_es_version()
+    if es_version < [7, 2, 0]:
+        pytest.skip("Cat Allocation metrics are only tested in version 7.2.0+")
```

Comment:
This can be removed since we don't test against older versions. See our e2e versions https://github.com/DataDog/integrations-core/blob/master/elastic/tox.ini#L6